cirq-core/cirq/sim/clifford/stabilizer_state_ch_form.py

```diff
@@ -236,17 +236,20 @@ def to_state_vector(self) -> np.ndarray:
 
         return arr
 
-    def _measure(self, q, prng: np.random.RandomState) -> int:
+    def _measure(self, q, prng: Optional[np.random.RandomState] = None) -> int:
         """Measures the q'th qubit.
 
         Reference: Section 4.1 "Simulating measurements"
 
         Returns: Computational basis measurement as 0 or 1.
         """
+        real_prng: np.random.RandomState = (
+            random_state.parse_random_state(np.random) if prng is None else prng
```

Comment:
Please revert this method.  `_measure()` is only called from the `measure()` method which can receive an integer seed argument.  Let's use `parse_random_state()` there.